readme.md

```diff
@@ -102,6 +132,10 @@ autoload -U promptinit; promptinit
 
 # optionally define some options
 PURE_CMD_MAX_EXEC_TIME=10
+# change path color
+zstyle :prompt:pure:path color white
+# prompt color whatever the result of the last command
+zstyle ':prompt:pure:prompt:*' green
```

Comment:
Hmm, I noticed this (star-glob) doesn't work if you also define `:prompt:pure:prompt:success` / `:error`. I'm not sure how `zstyle` decides on the precedence / weight of the settings.

Intuitively, if I have defined `zstyle :prompt:pure:prompt:success color white`, and later do `zstyle :prompt:pure:prompt:'*' color green`, then I think I'd want the glob to override the earlier statement. This might not be feasible / possible though, or maybe not even correct behavior.

If I did it the other way around, maybe I'd want the white to override the green `¯\_(ツ)_/¯`.

Do you have any thoughts on the topic?